site/layouts/_default/baseof.html

```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 <html lang="{{ $.Site.Language.Lang }}">
-  {{ partial "head.html" . }}
+  {{ partialCached "head.html" . }}
```

Comment:
This is showing a strange title for my home page. Do we need a more specific key to cache on?

Reply:
ah good point, forgot about the page title!

Reply:
I think I may just have to ditch the caching on this partial b/c the title can be different for each page.

Reply:
I realized we can cache the `course_banner.html` partial as well, because it will be the same for all pages w/in a course, so I added that

Reply:
Should this line still be `partialCached` if we're giving up on `head.html`?

Reply:
oh yeah, changed it in the other template but not here by accident